### PR TITLE
Feature/move dev to gov cloud

### PR DESCRIPTION
**Pull request by LindsayYoung**

### CODE REVIEW

**Comment by jontours** [comment]
Woo! tests pass, but they error locally.  I'll have to pair with y'all to figure out how to get cms tests to pass locally (is there something I'm doing wrong)

\cc @LindsayYoung @ccostino

**Comment by ccostino** [comment]
Ah just to avoid confusion @jontours and @LindsayYoung - this is the PR to merge into `develop` once our mini-release is done, but the branch in #801 needs to be merged into this branch first as a part of the work!

**Comment by ccostino** [comment]
`develop` should probably also be merged into here too prior to wrapping up to help account for any other conflicts (e.g., what I see with `fec/fec/settings/base.py` being listed right now).

**Comment by LindsayYoung** [comment]
If it is easier we can delete this branch and merge the other branch and reopen it. I don't want that branch to have develop merged into that branch because that is supposed to be on a code freeze for testing, if we need to cherry pick some commits, that is fine.

**Comment by ccostino** [comment]
Let's catch up briefly to make sure we have the right branches accounted for. :-)

**Comment by ccostino** [comment]
Merges are all done, this is ready for final review!

**Comment by ccostino** [comment]
Just confirmed that UUA/2FA stuff is included in this due to the ATO testing.

File: manifest_dev.yml [hunk 1]
```diff
@@ -6,8 +6,8 @@ services:
   - fec-creds-dev
   - fec-dev-cms
 env:
-  FEC_API_URL: https://fec-dev-api.18f.gov
-  FEC_APP_URL: https://fec-dev-proxy.18f.gov/data
+  FEC_API_URL: https://api-stage.open.fec.gov
```
Comment (ccostino) [inline manifest_dev.yml]: Whoops, this is the `dev` manifest but this is referencing staging.

File: tasks.py [hunk 2]
```diff
@@ -74,12 +74,14 @@ def _detect_space(repo, branch=None, yes=False):
 DEPLOY_RULES = (
     ('prod', _detect_prod),
     ('stage', lambda _, branch: branch.startswith('release')),
-    ('dev', lambda _, branch: branch == 'develop'),
+    # ('dev', lambda _, branch: branch == 'develop'),
+    # TODO take this out
+    ('stage', lambda _, branch: branch == 'feature/gov-cloud-support'),
```
Comment (ccostino) [inline tasks.py]: Ack, we'll have to fix this too.

File: manifest_dev.yml (hunk 1 above)
Comment (ccostino) [inline manifest_dev.yml]: @LindsayYoung do we have a dev subdomain setup for this like in the other spaces, or is it just `fec-dev-api.app.cloud.gov` for `dev`?

**Pull request merged**